styles.css

```diff
@@ -55,40 +55,40 @@
 .nav-ul {
   align-items: flex-start;
   list-style: none;
-  color: white;
   display: none;
   flex-direction: column;
-  background-color:#3C3A39;
-  width: 375px;
-  height: 755px;
+  background-color: #3c3a39;
+  width: 100%;
+  height: 570px;
   z-index: 100;
   position: absolute;
   left: 0;
-  padding-right: 35px;
-  padding-top: 108px;
+  padding-top: 110px;
   bottom: 0%;
   top: 0%;
+  overflow: hidden;
 }
 
-#envelope {
-  display: none;
+.nav-ul.active {
+  display: flex;
 }
```

Comment:
[OPTIONAL] Consider changing the font so it matches that of the template design.